spyder/plugins/history.py

```diff
@@ -205,37 +205,34 @@ def move_tab(self, index_from, index_to):
         """
         filename = self.filenames.pop(index_from)
         editor = self.editors.pop(index_from)
-        
+
         self.filenames.insert(index_to, filename)
         self.editors.insert(index_to, editor)
-        
+
     #------ Public API ---------------------------------------------------------
-    def add_history(self, filename):
+    def add_history(self, filename, history_list):

```

Comment:
`history_list` must be optional, so that history in the internal console can still work.